README.md

```diff
@@ -212,44 +197,66 @@ well as examples of downcasing (``http-api-data``). One choice for consistency
 
 ## Modules
 
-All publically facing modules (namely, those which are not listed in
-``other-modules`` in the Cabal file) MUST have explicit export lists.
+### Imports
 
 All modules MUST use one of the following conventions for imports:
 
-* ``import Foo (Baz, Bar, quux)``
-* ``import qualified Foo as F``
+* ``import Foo (Baz (Quux, quux), Bar, frob)``
+* ``import qualified Bar.Foo as Foo``
 
-If `ImportQualifiedPost` is enabled, the following form CAN also be used:
+More specifically, any imported module must _either_ explicitly name every
+identifier it imports, or be imported qualified. If `ImportQualifiedPost` is
```

Comment:
The *either* sounds too strong. I'm rather fond of having both

    import Data.Text (Text)
    import qualified Data.Text as Text

because I can't stand `Text.Text` in type signatures.

Reply:
Yeah, I agree this is a bit unclear. Your wording sounds better, so I'll use it.